### DISCUSSION TITLE
ACS and openSUSE

### DISCUSSION THREAD

**Discussion by tdtmusic2** [apache/cloudstack#9882] [Q&A] [answered]
Hi guys. Anyone here has a working cloud image source of openSUSE that works with ACS? In my case the 15.6 Leap Nocloud image does not boot, if I look at the console there's a blinking _ after Booting from Hard Drive. I tried to change the controller on the instance but it's the same. And if I think about it I had the same problem with some other cloud images, I think Arch or FreeBSD.

**Comment by kiranchavala** [comment]
@tdtmusic2 Are you able to boot the cloudimages  directly on the hypervisor ?

**Comment by kiranchavala** [comment]
@tdtmusic2 i have just tried opensuse 15.6

you need to select the installation option once the boot screen appears


![opensuse](https://github.com/user-attachments/assets/b018de4e-2a9b-4faf-8f48-fc2885166221)

https://www.reddit.com/r/openSUSE/comments/t9wop7/error_while_onstalling_opensuse_on_vmware/

**Reply by tdtmusic2** [reply]
> @tdtmusic2 i have just tried opensuse 15.6
> 
> you need to select the installation option once the boot screen appears
> 
> ![opensuse](https://private-user-images.githubusercontent.com/1401014/383468675-b018de4e-2a9b-4faf-8f48-fc2885166221.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA4ODI0NTcsIm5iZiI6MTczMDg4MjE1NywicGF0aCI6Ii8xNDAxMDE0LzM4MzQ2ODY3NS1iMDE4ZGU0ZS0yYTliLTRmYWYtOGY0OC1mYzI4ODUxNjYyMjEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTEwNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDExMDZUMDgzNTU3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MmE1NDNmNmYyODQ2OWE5OGRmNGQyMzcxOWNlZGM4M2Q3ZjQxMjcwNWE3MmRjZWJiMjVmZGIzNGFiNjUzZGQ2NiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.6DHRElkQZrMA-rmqdKHfPr9PRLPxBRROQEXI4DVSWHI)
> 
> https://www.reddit.com/r/openSUSE/comments/t9wop7/error_while_onstalling_opensuse_on_vmware/

You did not read my post. I can't get to any installation screen, it stops at the "BIOS" booting screen, with a blinking cursor.

**Comment by kiranchavala** [comment]
@tdtmusic2 

Could you please provide  more details of your environment 

1. Cloudstack version 
2. Hypervisor version details ( KVM/Vmware)

**Reply by tdtmusic2** [reply]
> @tdtmusic2
> 
> Could you please provide more details of your environment
> 
> 1. Cloudstack version
> 2. Hypervisor version details ( KVM/Vmware)

Sure. CS is 4.19.1.2, using it with KVM.

```
Name         : qemu-kvm
Epoch        : 17
Version      : 8.2.0
Release      : 11.el9_4.6
Architecture : x86_64

QEMU emulator version 8.2.0 (qemu-kvm-8.2.0-11.el9_4.6)
```

**Comment by kiranchavala** [comment]
@tdtmusic2 

I have tried with both 7.2.0 and 8.2.0 release of qemu-kvm  , but unable to reproduce the issue , the opensuse iso boots up fine 

Could you please provide the agent.log ( /var/log/cloudstack/agent/agent.log)  to investigate further

```
[root@server~]# yum info qemu-kvm
Last metadata expiration check: 1:15:47 ago on Wed 06 Nov 2024 10:18:27 AM UTC.
Installed Packages
Name         : qemu-kvm
Epoch        : 17
Version      : 7.2.0
Release      : 14.el9_2.5
Architecture : x86_64
Size         : 0.0
Source       : qemu-kvm-7.2.0-14.el9_2.5.src.rpm
Repository   : @System
From repo    : ol9_appstream
Summary      : QEMU is a machine emulator and virtualizer
URL          : http://www.qemu.org/
License      : GPLv2 and GPLv2+ and CC-BY
Description  : qemu-kvm is an open source virtualizer that provides hardware
             : emulation for the KVM hypervisor. qemu-kvm acts as a virtual
             : machine monitor together with the KVM kernel modules, and emulates the
             : hardware for a full system such as a PC and its associated peripherals.
             
```
             
```
 [root@server~]# yum info qemu-kvm
Last metadata expiration check: 1:18:42 ago on Wed 06 Nov 2024 10:18:27 AM UTC.
Installed Packages
Name         : qemu-kvm
Epoch        : 17
Version      : 8.2.0
Release      : 11.el9_4.6
Architecture : x86_64
Size         : 0.0
Source       : qemu-kvm-8.2.0-11.el9_4.6.src.rpm
Repository   : @System
From repo    : ol9_appstream
Summary      : QEMU is a machine emulator and virtualizer
URL          : http://www.qemu.org/
License      : GPLv2 and GPLv2+ and CC-BY
Description  : qemu-kvm is an open source virtualizer that provides hardware
             : emulation for the KVM hypervisor. qemu-kvm acts as a virtual
             : machine monitor together with the KVM kernel modules, and emulates the
             : hardware for a full system such as a PC and its associated peripherals.

```

**Reply by tdtmusic2** [reply]
> @tdtmusic2
> 
> I have tried with both 7.2.0 and 8.2.0 release of qemu-kvm , but unable to reproduce the issue , the opensuse iso boots up fine
> 
> Could you please provide the agent.log ( /var/log/cloudstack/agent/agent.log) to investigate further
> 
> ```
> [root@server~]# yum info qemu-kvm
> Last metadata expiration check: 1:15:47 ago on Wed 06 Nov 2024 10:18:27 AM UTC.
> Installed Packages
> Name         : qemu-kvm
> Epoch        : 17
> Version      : 7.2.0
> Release      : 14.el9_2.5
> Architecture : x86_64
> Size         : 0.0
> Source       : qemu-kvm-7.2.0-14.el9_2.5.src.rpm
> Repository   : @System
> From repo    : ol9_appstream
> Summary      : QEMU is a machine emulator and virtualizer
> URL          : http://www.qemu.org/
> License      : GPLv2 and GPLv2+ and CC-BY
> Description  : qemu-kvm is an open source virtualizer that provides hardware
>              : emulation for the KVM hypervisor. qemu-kvm acts as a virtual
>              : machine monitor together with the KVM kernel modules, and emulates the
>              : hardware for a full system such as a PC and its associated peripherals.
>              
> ```
> 
> ```
>  [root@server~]# yum info qemu-kvm
> Last metadata expiration check: 1:18:42 ago on Wed 06 Nov 2024 10:18:27 AM UTC.
> Installed Packages
> Name         : qemu-kvm
> Epoch        : 17
> Version      : 8.2.0
> Release      : 11.el9_4.6
> Architecture : x86_64
> Size         : 0.0
> Source       : qemu-kvm-8.2.0-11.el9_4.6.src.rpm
> Repository   : @System
> From repo    : ol9_appstream
> Summary      : QEMU is a machine emulator and virtualizer
> URL          : http://www.qemu.org/
> License      : GPLv2 and GPLv2+ and CC-BY
> Description  : qemu-kvm is an open source virtualizer that provides hardware
>              : emulation for the KVM hypervisor. qemu-kvm acts as a virtual
>              : machine monitor together with the KVM kernel modules, and emulates the
>              : hardware for a full system such as a PC and its associated peripherals.
> ```

I think you're using the desktop version of openSUSE. I specifically said the cloud image has this issue, in my case is openSUSE-Leap-15.6.aarch64-1.0.1-NoCloud-Build1.86.qcow2.

**Comment by kiranchavala** [comment, marked as answer]
Hi @tdtmusic2 

I just tried it with  "openSUSE-Leap-15.6.x86_64-1.0.1-NoCloud-Build1.92.qcow2 " and I was able to boot 

https://download.opensuse.org/repositories/Cloud:/Images:/Leap_15.6/images/openSUSE-Leap-15.6.x86_64-1.0.1-NoCloud-Build1.92.qcow2

<img width="1054" alt="Screenshot 2024-11-06 at 6 55 11 PM" src="https://github.com/user-attachments/assets/d301148e-eaae-4265-892a-4ca016cd5393">

**Reply by tdtmusic2** [reply]
> Hi @tdtmusic2
> 
> I just tried it with "openSUSE-Leap-15.6.x86_64-1.0.1-NoCloud-Build1.92.qcow2 " and I was able to boot
> 
> https://download.opensuse.org/repositories/Cloud:/Images:/Leap_15.6/images/openSUSE-Leap-15.6.x86_64-1.0.1-NoCloud-Build1.92.qcow2
> 
> <img alt="Screenshot 2024-11-06 at 6 55 11 PM" width="1054" src="https://private-user-images.githubusercontent.com/1401014/383570205-d301148e-eaae-4265-892a-4ca016cd5393.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA5MDEzMTAsIm5iZiI6MTczMDkwMTAxMCwicGF0aCI6Ii8xNDAxMDE0LzM4MzU3MDIwNS1kMzAxMTQ4ZS1lYWFlLTQyNjUtODkyYS00Y2EwMTZjZDUzOTMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTEwNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDExMDZUMTM1MDEwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZjcxMGU2YjU3NjM1YTYzYjIyY2Q3NDZlMGMwZjIzMzM4MDM2OWFiZDA0ODRkZTNmYzA5ODJjYzE5YTU5OGZlNiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.BrVFRY5dupx5wLz4X4pgAaICjLjmZuPXa5ebaz9YxSA">

Mine is like this.
![Screenshot 2024-11-06 154956](https://github.com/user-attachments/assets/dc582951-3e44-4f38-bab5-eb54ff163b33)
EDIT: Nevermind, I'm blind...I was using the aarch64 image instead of x86_64. Sorry for the confusion :(
